assignment02/js/myScript2.js

```diff
@@ -0,0 +1,36 @@
+$(document).ready(function(){

```

Comment:
Very nice and clean! Great formatting and easy to read - all in an anon function!